modules/rosa-sts-cluster-terraform-file-creation.adoc

```diff
@@ -321,100 +438,89 @@ variable "aws_region" {
 }
 EOF
 ----
+ifndef::tf-defaults[]
 
-. To set up Terraform to create your resources based on your Terraform files, run the following command:
+. Create the `vpc.tf` file by running the following command:
 +
 [source,terminal]
 ----
-$ terraform init
-----
+$ cat<<-EOF>vpc.tf
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "5.1.2"
 
-. Optional: Verify that the Terraform you copied is correct by running the following command:
-+
-[source,terminal]
-----
-$ terraform validate
+  count = var.create_vpc ? 1 : 0
+  name  = var.vpc_name
+  cidr  = var.vpc_cidr_block
+
+  azs             = var.availability_zones
+  private_subnets = var.private_subnet_cidrs
+  public_subnets  = var.public_subnet_cidrs
+
+  enable_nat_gateway   = true
+  single_nat_gateway   = var.single_nat_gateway
+  enable_dns_hostnames = true
+  enable_dns_support   = true
+
+  tags = var.additional_tags
+}
+EOF
 ----
+
+. Optional: Create the `terraform.tfvars` file by running the following command:
 +
-.Sample output
+[NOTE]
+====
+You use the `terraform.tfvars` file to change your variables in one place without touching the rest of your Terraform files. If you do not create a `terraform.tfvars` file, you are prompted for the required variables during cluster creation.
+====
```

Comment:
Also add the note here:
[NOTE]
====
Copy and then edit this file *before* running the command to customize your <list whatever can be customized in this file>.
====
But I don't love the idea of two notes in row. So we may need to fix that.